lib/multiplexer/multiplexer.go

```diff
@@ -500,6 +500,13 @@ func (m *Mux) detect(conn net.Conn) (*Conn, error) {
 					}).Warnf("%s - could not get host CA", invalidProxySignatureError)
 					continue
 				}
+				if errors.Is(err, ErrNonLocalCluster) {
+					m.WithFields(log.Fields{
+						"src_addr": conn.RemoteAddr(),
+						"dst_addr": conn.LocalAddr(),
+					}).Debugf("%s - signed by non local cluster", invalidProxySignatureError)
```

Comment:
Should this log level be higher?  This seems like a `warning` condition to me

Reply:
Users who changed cluster name would have a lot of these warnings and they can't really do anything about it, and understanding it also is not simple. It's more of an error level, we just don't have a proper fix for it yet.